composer/core/types.py

```diff
@@ -61,14 +63,14 @@
 ]
 
 T = TypeVar('T')
-Many = Union[T, Tuple[T, ...], List[T]]
+Many = Union[T, Sequence[T]]
 
 Tensor = torch.Tensor
-Tensors = Many[Tensor]
+Tensors = Union[Tensor, Sequence[Tensor]]
```

Comment:
Same thing...removing the aliases here will ensure that we removed them throughout the entire codebase, and use the underlying `Union[X, Sequence[X]]` notation in the codebase throughout.